libevmasm/AssemblyItem.h

```diff
@@ -92,13 +97,30 @@ class AssemblyItem
 		m_debugData{langutil::DebugData::create()}
 	{}
 
+	static AssemblyItem eofCreate(ContainerID _containerID, langutil::DebugData::ConstPtr _debugData = langutil::DebugData::create())
+	{
+		return AssemblyItem(EOFCreate, _containerID, std::move(_debugData));
+	}
+	static AssemblyItem returnContract(ContainerID _containerID, langutil::DebugData::ConstPtr _debugData = langutil::DebugData::create())
+	{
+		return AssemblyItem(ReturnContract, _containerID, std::move(_debugData));
+	}
+	static AssemblyItem jumpTo(AssemblyItem _tag, langutil::DebugData::ConstPtr _debugData = langutil::DebugData::create())
```

Comment:
Since this is specifically for `RJUMP` and not `JUMP`, I think you should include "relative" in the name. Same for `RJUMPI`.

```suggestion
	static AssemblyItem relativeJumpTo(AssemblyItem const& _tag, langutil::DebugData::ConstPtr _debugData = langutil::DebugData::create())
```

Also, you're copying the whole `_tag`. Shouldn't you pass a reference instead?

`_debugData` should also be moved rather than copied.

Reply:
You should also assert that `_tag` is actually one of the item types that are allowed to contain a tag and not some other random type with unrelated data.

Reply:
Done